packages/http-client-csharp/generator/TestProjects/CadlRanch/http/routes/tspCodeModel.json

```diff
@@ -0,0 +1,3453 @@
+    {
+     "$id": "296",
+     "Name": "endpoint",
+     "NameInRequest": "endpoint",
+     "Description": "Service host",
+     "Type": {
+      "$id": "297",
+      "kind": "url",
+      "name": "url",
+      "crossLanguageDefinitionId": "TypeSpec.url"
+     },
+     "Location": "Uri",
+     "IsApiVersion": false,
+     "IsResourceParameter": false,
+     "IsContentType": false,
+     "IsRequired": true,
+     "IsEndpoint": true,
+     "SkipUrlEncoding": false,
+     "Explode": false,
+     "Kind": "Client",
+     "DefaultValue": {
+      "$id": "298",
+      "Type": {
+       "$id": "299",
+       "kind": "string",
+       "name": "string",
+       "crossLanguageDefinitionId": "TypeSpec.string"
+      },
+      "Value": "http://localhost:3000"
+     }
+    }
+   ],
+   "Decorators": []
+  },
+  {
+   "$id": "300",
+   "Name": "QueryParametersQueryContinuation",
+   "Operations": [],
+   "Protocol": {
+    "$id": "301"
+   },
+   "Parent": "QueryParameters",
+   "Parameters": [
+    {
+     "$id": "302",
+     "Name": "endpoint",
+     "NameInRequest": "endpoint",
+     "Description": "Service host",
+     "Type": {
+      "$id": "303",
+      "kind": "url",
+      "name": "url",
+      "crossLanguageDefinitionId": "TypeSpec.url"
+     },
+     "Location": "Uri",
+     "IsApiVersion": false,
+     "IsResourceParameter": false,
+     "IsContentType": false,
+     "IsRequired": true,
+     "IsEndpoint": true,
+     "SkipUrlEncoding": false,
+     "Explode": false,
+     "Kind": "Client",
+     "DefaultValue": {
+      "$id": "304",
+      "Type": {
+       "$id": "305",
+       "kind": "string",
+       "name": "string",
+       "crossLanguageDefinitionId": "TypeSpec.string"
+      },
+      "Value": "http://localhost:3000"
+     }
+    }
+   ],
+   "Decorators": []
+  },
+  {
+   "$id": "306",
+   "Name": "QueryParametersQueryContinuationStandard",
+   "Operations": [
+    {
+     "$id": "307",
+     "Name": "primitive",
+     "ResourceName": "Standard",
+     "Accessibility": "public",
+     "Parameters": [
+      {
+       "$id": "308",
+       "Name": "param",
+       "NameInRequest": "param",
+       "Type": {
+        "$id": "309",
+        "kind": "string",
+        "name": "string",
+        "crossLanguageDefinitionId": "TypeSpec.string",
+        "decorators": []
+       },
+       "Location": "Query",
+       "IsApiVersion": false,
+       "IsContentType": false,
+       "IsEndpoint": false,
+       "Explode": false,
+       "IsRequired": true,
+       "Kind": "Method",
+       "Decorators": [],
+       "SkipUrlEncoding": false
+      }
+     ],
+     "Responses": [
+      {
+       "$id": "310",
+       "StatusCodes": [
+        204
+       ],
+       "BodyMediaType": "Json",
+       "Headers": [],
+       "IsErrorResponse": false
+      }
+     ],
+     "HttpMethod": "GET",
+     "RequestBodyMediaType": "None",
+     "Uri": "{endpoint}",
+     "Path": "/routes/query/query-continuation/standard/primitive?fixed=true",
```

Comment:
This path is not valid since it includes the query parameter. This would need to be fixed upstream in the emitter or TCGC. "fixed" should be a parameter with a literal value type.